operators/falcon/falcon.package.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
```

Comment:
@isimluk please remove this file. It is not needed for bundle format

Reply:
Thank You, @mvalarh